bin/steps/appdynamics-setup

```diff
@@ -0,0 +1,76 @@
+BUILD_DIR='/app'
+echo "Setting up for Appdynamics"
+
+if ! grep -Fxq "appdynamics" /app/requirements.txt
+then
+  echo "appdynamics" >> /app/requirements.txt
```

Comment:
We'd recommend against altering the user's requirements.txt file as it can lead to confusing, unexpected behavior.